templates/traefik/traefik.toml

```diff
@@ -204,4 +204,6 @@ onDemand = false # create certificate when container is created
           "thelounge.{{ ansible_nas_domain }}",
           "transmission.{{ ansible_nas_domain }}",
           "transmission-openvpn.{{ ansible_nas_domain }}",
-          "znc.{{ ansible_nas_domain }}"]
+          "znc.{{ ansible_nas_domain }}",
+          "joomla.{{ ansible_nas_domain }}"
```

Comment:
You may (or may not) have noticed that the rest of the hostnames are alphabetical :)